CHANGES.md

```diff
@@ -29,11 +29,19 @@ To be released.
     given.
  -  Fixed a bug that TURN relay had been disconnected when being connected for longer than 5
     minutes.
+ -  `BlockChain<T>` now implements `IReadOnlyList<Block<T>>`.  [[#205]]
+    `BlockChain<T>.Validate()` now receives `IReadOnlyList<Block<<T>>`
+    instead of `IEnumerable<Block<T>>`.  [[#205]]
+ -  `IBlockPolicy<T>.ValidateBlocks()` and
+    `IBlockPolicy<T>.GetNextBlockDifficulty()`
+    now receives `IReadOnlyList<Block<<T>>` instead of `IEnumerable<Block<T>>`.
```

Comment:
Plural present tense.

```suggestion
    now receive `IReadOnlyList<Block<<T>>` instead of `IEnumerable<Block<T>>`.
```

As a non-native speaker, I prefer to use “became to do” instead of “now does/do”, because in the latter way a verb should inflect depending on its subject (singular or plural).  On the other hand, a verb following “became to do” is always singular present, so less prone to make a grammatical mistake.  🙄